core/arch/arm/include/mm/core_mmu.h

```diff
@@ -141,6 +143,7 @@ static inline const char *teecore_memtype_name(enum teecore_memtypes type)
 		[MEM_AREA_TEE_RAM_RW] = "TEE_RAM_RW",
 		[MEM_AREA_NEX_RAM_RW] = "NEX_RAM_RW",
 		[MEM_AREA_TEE_ASAN] = "TEE_ASAN",
+		[MEM_AREA_IDENTITY_MAP_RX] = "MEM_AREA_IDENTITY_MAP_RX",
```

Comment:
Sorry I missed that in my review... should be `= "IDENTITY_MAP_RX"`

Reply:
I'll fix